Expensify/App
# Revert "Merge pull request #4085 from Expensify/chirag-revert-3979"

This reverts commit c2f92b63d2e8f43310162ccddea32446cbbd5753. 


@chiragsalian this a PR related to our discussion here https://github.com/Expensify/Expensify.cash/pull/3979#issuecomment-880938773

### Details


### Fixed Issues

$ #3743
$ #3293

### Tests / QA Steps
 


1. Send a message. You should see it in LHN.
2. Delete that message. The LHN should say "[Comment deleted]".
3. Refresh the page. Wait a few seconds. The LHN should still say "[Comment deleted]".
4. Send IOU. You should see "Requested €123 from A" or something like that in LHN.
5. Refresh the page. Wait a few seconds. It should still say "Requested €123 from A" and not "[Comment deleted]". (This one requires #3737 to be fixed).

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/64093836/125278225-20de6500-e31b-11eb-8a8b-712270ebf141.mp4


#### Mobile Web


https://user-images.githubusercontent.com/64093836/125286644-b9c5ae00-e324-11eb-947a-ed746b6f4369.mp4


#### Desktop


https://user-images.githubusercontent.com/64093836/125287448-97806000-e325-11eb-8dd0-05ef737ba620.mov


#### iOS



https://user-images.githubusercontent.com/64093836/125287700-eaf2ae00-e325-11eb-9ae3-db1d93a26886.mp4



#### Android



https://user-images.githubusercontent.com/64093836/125279106-343e0000-e31c-11eb-8421-efde79909afc.mp4
